packages/language-server/api.d.ts

```diff
@@ -25,6 +25,13 @@ export type ServerInitializationOptions = {
   logLevel?: LogLevel;
 };
 
+export function getNodeName(
```

Comment:
why does the language server exposes a logical utility function?
It generally runs in a different process and communicates via JSON_RPC

Reply:
Generally the language server should use logical utilities from other packages.
Maybe getNodeName should be extracted out of the language server sub-package.